packages/flutter/lib/src/material/chip_action.dart

```diff
@@ -145,12 +165,87 @@ class ActionChip extends StatelessWidget implements ChipAttributes, TappableChip
       clipBehavior: clipBehavior,
       focusNode: focusNode,
       autofocus: autofocus,
+      disabledColor: disabledColor,
       padding: padding,
       visualDensity: visualDensity,
+      isEnabled: isEnabled,
       labelPadding: labelPadding,
       materialTapTargetSize: materialTapTargetSize,
       elevation: elevation,
       shadowColor: shadowColor,
+      surfaceTintColor: surfaceTintColor,
     );
   }
 }
+
+// BEGIN GENERATED TOKEN PROPERTIES
+
+// Generated code to the end of this file. Do not edit by hand.
+// These defaults are generated from the Material Design Token
+// database by the script dev/tools/gen_defaults/bin/gen_defaults.dart.
+
+// Generated version v0_101
+class _TokenDefaultsM3 extends ChipThemeData {
+  const _TokenDefaultsM3(this.context, this.isEnabled)
+    : super(
+        elevation: 0.0,
+        shape: const RoundedRectangleBorder(borderRadius: BorderRadius.only(topLeft: Radius.circular(8.0), topRight: Radius.circular(8.0), bottomLeft: Radius.circular(8.0), bottomRight: Radius.circular(8.0))),
```

Comment:
Could be `BorderRadius.all(Radius.circular(8))`

Reply:
Yup, this is a generator problem. One that will be fixed with the BottomAppBar PR.